### DISCUSSION TITLE
Mantine CSS variables are not defined

### DISCUSSION THREAD

**Discussion by PhanDungTri** [mantinedev/mantine#1466] [Q&A] [answered]
I'm customizing the `Button` component and using Mantine CSS variables. But the browser said that the variables are not defined!

![image](https://user-images.githubusercontent.com/26342804/168484065-7ff379c5-81c8-48a3-bdf1-4f8f2bb28f25.png)

Any idea with this problem? Thank you.

**Comment by rtivital** [comment, marked as answer]
Have you set `withCSSVariables` on MantineProvider? Have you updated `@mantine/*` packages to latest version?
